pkg/foxtrot/foxtrot.go

```diff
@@ -11,3 +13,21 @@ type User struct {
 	passwordHash string
 	avatar       []byte
 }
+
+// Room is a chat room identified by its name.
+type Room struct {
+	Name string `json:"name"`
+}
+
+// Message is a chat message.
+type Message struct {
+	ID        int    `json:"id"` // ordered by creation time
+	Content   string `json:"content"`
+	CreatedAt string `json:"createdAt"`
+	Room      string `json:"room"`
+	Author    string `json:"author"`
+}
+
+func now() string {
```

Comment:
maybe this should be in a package called `foxtime` so we can say "It's fox time, now!" ;-)

Reply:
🍭